### DISCUSSION TITLE
bao hypersior can support multi linux guest os

### DISCUSSION THREAD

**Discussion by wanlon** [bao-project/bao-hypervisor#131] [General]
Hi,
     Can bao hypersior support multi linux guest os running? If supports, what should I do for this.

**Comment by josecm** [comment]
@wanlon there should be no isssue in running two linux instances. We have done it multiple times.

The only "issue" you may run into is using different outputs for each linux image. Depending on your platform, for this I suggest using an UART device for one instance and an ethernet for another, as we do for the freertos+linux configurations in bao-demos. Of course you will need two separate device trees for each instance that match each VM configuration.

If can you give me more details on your setup, maybe I can help you further.

**Comment by wanlon** [comment]
Hello, our actual hardware platform is NXP's s32g399 chip. Currently, we want to simulate arm64 on QEMU and run multiple Linux guest OSs simultaneously

**Comment by danielRep** [comment]
I moved this to a discussion since there is no direct issue related to the question.
